build.gradle

```diff
@@ -2,7 +2,7 @@ ext {
     coroutinesVersion = '1.5.2'
     wordPressUtilsVersion = '2.4.0'
     wordPressLoginVersion = 'trunk-a229c42dc36badcc39f7be3d3ad5231bb031eee9'
-    gutenbergMobileVersion = 'v1.71.1'
+    gutenbergMobileVersion = '4615-4a8ef81f3e79ffbe72b2040b04afb689cff18891'
```

Comment:
Should we update this to an alpha tag before merging?

Reply:
Yeah, once we merge the GB-mobile PR, we should update the reference to point to an alpha tag.

Reply:
Gutenberg Mobile reference updated with `v1.72.0-alpha3` tag.